Include/Acidanthera/Library/OcConfigurationLib.h

```diff
@@ -407,11 +407,28 @@ typedef enum {
   OC_ARRAY (OC_MISC_TOOLS_ENTRY, _, __)
   OC_DECLARE (OC_MISC_TOOLS_ARRAY)
 
+#define OC_MISC_SERIAL_FIELDS(_, __) \
```

Comment:
Is it possible to directly use these values from https://github.com/acidanthera/audk/blob/bb1bba3d776733c41dbfa2d1dc0fe234819a79f2/MdeModulePkg/MdeModulePkg.dec?

If not, we should create some macros ourselves then I believe.

Reply:
I don't know. To understand how the UEFI build system works, I would add `-save-temps` to the GCC and Xcode compiler flags in the OpenCorePkg.dsc file. The temp files are stored in the `Build` directory of `UDK`. Then look at any `.i` file that includes OcConfigurationLib.h to see if you can find the Pcd values or variables.

Reply:
Well, then it's not possible. Let's add them to OcMainLib.h maybe?

Reply:
UPDATE - This is not possible. Instead, we just copy-paste the values from MdeModulePkg.dec.